CrazyCanvas/Include/Multiplayer/SingleplayerInitializer.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "LambdaEngine.h"
+
+namespace LambdaEngine
+{
+	class SingleplayerInitializer
```

Comment:
Is the class going to have member variables in the future? Otherwise there's no reason to make this a class.